_reports/GuangyuanYang.md

```diff
@@ -164,8 +164,8 @@ investigation/review. Based on this feature, I implemented a
 [script](https://gist.github.com/yzgyyang/13ebb46a5c90ac624b5c6eb43deedc0d) to
 demonstrate the ability of bootstrapping coala Go bears on a bare-metal FreeBSD
 system
-([Screen record for the demo](https://asciinema.org/a/0poVbHE7lqehV4islFo6cptjb)),
-and it can be expanded to support multiple platforms.
+([Screen record for the demo]()),and it can be expanded to support multiple
```

Comment:
this is invalid syntax for the link, and `,and` is bad grammar.

Also we want this link even though it is broken, as part of the record of this project.